### PR TITLE
IA-671: ignore source if empty, add api params to ignore empty names

### PR DESCRIPTION
**Pull request by beygorghor**

### CODE REVIEW

**Review by quang-le — APPROVED** [review]
Front-end looks fine to me

File: hat/assets/js/apps/Iaso/domains/orgUnits/utils.js [hunk 1]
```diff
@@ -66,9 +66,14 @@ export const getSourcesWithoutCurrentSource = (
...
-        message = `${orgUnit.name} - source: ${orgUnit.source}`;
-        if (orgUnit.org_unit_type_name && withType) {
-            message += ` (${orgUnit.org_unit_type_name})`;
+        if (orgUnit.name) {
+            message = `${orgUnit.name}${
+                orgUnit.source ? ` - source: ${orgUnit.source}` : ''
+            }${
+                orgUnit.org_unit_type_name && withType
+                    ? ` (${orgUnit.org_unit_type_name})`
+                    : ''
```
Comment (olethanh) [inline hat/assets/js/apps/Iaso/domains/orgUnits/utils.js:75]: it's like an  escape in an escape in a escape. I've got a feeling there is probably a more readable way to write this?

Comment (olethanh) [reply hat/assets/js/apps/Iaso/domains/orgUnits/utils.js:75]: maybe something like
```
orgUnit.name \
   + orgunit.source ? ` - source: ${orgUnit.source}` : '' \
   + orgUnit.org_unit_type_name && withType?  ` (${orgUnit.org_unit_type_name})` : ''
```

**Review by olethanh — CHANGES_REQUESTED** [review]
We should probably forbid empty name in the futur but that's probably a good fix in the mean time

File: hat/assets/js/apps/Iaso/domains/orgUnits/utils.js [hunk 2]
```diff
@@ -63,18 +63,19 @@ export const getSourcesWithoutCurrentSource = (
     return sources;
 };
 
-export const getOrgunitMessage = (orgUnit, withType) => {
+export const OrgunitMessage = ({ orgUnit, withType }) => {
     let message = textPlaceholder;
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/orgUnits/utils.js:67]: name should be orgunitMessage, since it's not a class

**Pull request merged**